server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/test/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutesTest.java

```diff
@@ -1892,4 +2143,18 @@ private DeletedMessage storeDeletedMessage(DeletedMessage deletedMessage) {
             .block();
         return deletedMessage;
     }
+
+    private DeletedMessage generateNoExpiredDeletedMessage() {
```

Comment:
This is used in one single place. Then please don't hide this nice builder behing a cryptic method name...